src/index.html

```diff
@@ -7,8 +7,51 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header class="header">
+      <a href="#home">
+        <img
+          src="./images/logo.png"
+          alt="logo company"
+          class="image"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
```

Comment:
```suggestion
            <a href="#apple" class="nav__link is-active">Apple</a>
```
add more specific link names

Reply:
What means specific?